src/hotspot/cpu/x86/stubGenerator_x86_64_aes.cpp

```diff
@@ -255,23 +287,25 @@ address StubGenerator::generate_galoisCounterMode_AESCrypt() {
 #ifdef _WIN64
   // on win64, fill len_reg from stack position
   __ push(rsi);
+  __ push(rdi);
   __ movptr(key, key_mem);
   __ movptr(state, state_mem);
 #endif
   __ movptr(subkeyHtbl, subkeyH_mem);
   __ movptr(counter, counter_mem);
 // Align stack
   __ andq(rsp, -64);
-  __ subptr(rsp, 96 * longSize); // Create space on the stack for htbl entries
+  __ subptr(rsp, 200 * longSize); // Create space on the stack for htbl entries
```

Comment:
It will be helpful to document the structure of subkeyHtbl (what is there in the 200 long entries).